src/04-modality-specific-files/02-magnetoencephalography.md

```diff
@@ -630,4 +623,28 @@ A guide for using macros can be found at
    }
 ) }}
 
+Example 2:
+
+<!-- This block generates a file tree.
+A guide for using macros can be found at
+ https://github.com/bids-standard/bids-specification/blob/master/macros_doc.md
+-->
+{{ MACROS___make_filetree_example(
```

Comment:
My suggestion is just to add a note here that the empty room recordings can be symlinked and note that it might be helpful in the case where two empty room recordings correspond to the same subject to save data space

Reply:
Not everyone might know who is reading this spec without looking it up that symlinks are allowed in BIDS

Reply:
@alexrockhill I'd be against adding this here, this is really an "implementation detail" that won't even be platform-agnostic and portable. If a Linux user zips a BIDS structure containing symlinks and sends it to their Windows friends, they'll often into issues (depending on how the zip file was created)

Let's open a separate discussion on the very issue of symlinks, no? :)

Reply:
Sounds good

Reply:
are symbolic links allowed? I searched the specification, the word "symlink" or "symbolic" is nowhere to be found. Symbolic links would indeed not work on all file systems and operating systems. Great if they work for your use-case, but  I am not sure whether they should be included in the specification. 

Please do start a separate issue so that non-MEG folks can chime in as well. It would not surprise me if this has already been discussed in the past and that someone can share insights from the past.